packages/core/admin/admin/src/pages/Settings/pages/ApiTokens/ListView.tsx

```diff
@@ -198,10 +191,9 @@ export const ListView = () => {
         }
       />
       <ContentLayout>
-        {!canRead && <NoPermissions />}
-        {canRead && apiTokens.length > 0 && (
+        {apiTokens.length > 0 && (
```

Comment:
can you please explain why you removed canRead permission at many places and its present in few cases.

Reply:
I was thinking that you don't need to see the page if you can't read, but on reflection, you can still create them 🤔 so we probably need to revert the changes here... and potentially make them consistent across the rest of the settings sections 🙃

Reply:
following up on this, it's also inconsistent across the app... some settings have a `main` permission whilst things like users & roles don't...